src/app/utils/testing.utils.ts

```diff
@@ -79,6 +82,25 @@ function mockApplicantDetails(args?: Partial<ApplicantDetails>): ApplicantDetail
   return { ...mock, ...args };
 }
 
+function mockDDA(args?: Partial<DDA>): DDA {
+  const mock: DDA = {
```

Comment:
Reduce this to some essential keys and pass the remaining fields if relevant to the test via the `it` test